src/widgets/Notebook.cpp

```diff
@@ -104,6 +105,20 @@ Notebook::Notebook(QWidget *parent)
             << "Notebook must be created within a BaseWindow";
     }
 
+    this->sortTabsAlphaAction_ = new QAction("Sort Tabs Alphabetically", this);
+
+    QObject::connect(this->sortTabsAlphaAction_, &QAction::triggered, [this] {
+        if (!this->isNotebookLayoutLocked())
```

Comment:
It could be confusing that nothing happens if you trigger this when the layout is locked.

We should make sure this can't be triggered from the emote popup. Though, I think that's already the case.

Reply:
That's great! In my view, the locked notebook is for preventing accidental drags and here, the user explicitly clicked on the button.